velox/functions/prestosql/tests/ArrayHasDuplicatesTest.cpp

```diff
@@ -157,8 +157,7 @@ TEST_F(ArrayHasDuplicatesTest, json) {
   result = evaluate(
       "array_has_duplicates(C0)",
       makeRowVector({makeNullableArrayVector<StringView>(
-          {{R"({"key":"value"})", R"({"key":"another_value"})"},
```

Comment:
I dont see anything wrong here, so why does this fail on the linux release build but passes on ubuntu ?